osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -99,10 +100,6 @@ public ButtonSystem()
             buttonsTopLevel.Add(new Button(@"play", @"button-play-select", FontAwesome.fa_osu_logo, new Color4(102, 68, 204, 255), () => State = ButtonSystemState.Play, WEDGE_WIDTH, Key.P));
             buttonsTopLevel.Add(new Button(@"osu!editor", @"button-generic-select", FontAwesome.fa_osu_edit_o, new Color4(238, 170, 0, 255), () => OnEdit?.Invoke(), 0, Key.E));
             buttonsTopLevel.Add(new Button(@"osu!direct", @"button-direct-select", FontAwesome.fa_osu_chevron_down_o, new Color4(165, 204, 0, 255), () => OnDirect?.Invoke(), 0, Key.D));
```

Comment:
i'd probably just move everything to load along with the exit button you removed.

Reply:
Is it worth moving everything from the `MainMenu` constructor into `load` too?

Reply:
if everything still works definitely. it's generally where we prefer more complex initialisation to be as there's more potential for async.